scripts/vscodeserver/Project.toml

```diff
@@ -0,0 +1,15 @@
+name = "vscodeserver"
```

Comment:
Maybe `VSCodeServer`? That seems more in line with the package naming guidelines.

Reply:
yeah, sounds good. Do we want to move `vscodeserver` directly into `scripts/packages` ?

Reply:
Hm, good question. So far I had thought of the `packages` folder as stuff that is git submodules, which I think we can avoid for this story here, right? Probably doesn't matter much in the end.

Reply:
agreed.